tests/caasp/oci_keyboard.pm

```diff
@@ -20,6 +20,7 @@ sub run {
     check_var('VIDEOMODE', 'text') ? send_key 'alt-y' : send_key 'alt-e';
     send_key 'up';
     send_key 'ret' if check_var('VIDEOMODE', 'text');
+    sleep 1;
```

Comment:
why do you need `sleep`? is there something like a *race* condition?

Reply:
I added explanation